### PR TITLE
OSDOCS-7644

### PR DESCRIPTION
**Pull request by smunje1**
[OSDOCS-7644](https://issues.redhat.com/browse/OSDOCS-7644): [Doc] SRE STS AWS view

Aligned team: Service Delivery
OCP version for cherry-picking: enterprise-4.13+
JIRA issues: [OSDOCS-7644](https://issues.redhat.com/browse/OSDOCS-7644)
Preview pages: [ROSA build preview](https://64405--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix#rosa-sre-sts-view-aws-account_rosa-policy-responsibility-matrix)
SME review **completed**: @paulczar and @jaybeeunix
QE review: **NA**
Peer review **completed**: @bmcelvee

**Note**: We are including only ROSA content and not OSD content as part of Acceptance Criteria for this Jira. For more information, please see [slack thread](https://redhat-internal.slack.com/archives/C05Q2MLFB0X/p1694093293402269?thread_ts=1694003063.601879&cid=C05Q2MLFB0X).

### CODE REVIEW

File: modules/policy-identity-access-management.adoc [hunk 1]
```diff
@@ -49,8 +49,10 @@ In AWS, SREs generate a short-lived AWS access token for the `BYOCAdminAccess` u
 
 In Google Cloud, SREs access resources after being authenticated against a Red Hat SAML identity provider (IDP). The IDP authorizes tokens that have time-to-live expirations. The issuance of the token is auditable by corporate Red Hat IT and linked back to an individual user.
 
+include::osd-sre-sts-view-aws-account.adoc[leveloffset=+1]
```
Comment (kalexand-rh) [inline modules/policy-identity-access-management.adoc:52]: @smunje1 , you can't include a module in a module.

Comment (kalexand-rh) [reply modules/policy-identity-access-management.adoc:52]: If you don't need the heading but do need it in the middle of the module, consider transforming it into into a [snippet](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-text-snippets).

If you aren't expecting to use it in more than one location, include the text directly.

Comment (smunje1) [reply modules/policy-identity-access-management.adoc]: Ack. Thank you!

File: modules/policy-identity-access-management.adoc [hunk 2]
```diff
@@ -49,8 +49,19 @@ In AWS, SREs generate a short-lived AWS access token for the `BYOCAdminAccess` u
...
+[id="osd-sre-sts-view-aws-account_{context}"]
+== SRE STS view of AWS accounts
+
+When SRE is on VPN through two-factor authentication, Red Hat Support and SRE can assume the `ManagedOpenShift-Support-Role` in your AWS Account. `ManagedOpenShift-Support-Role` has all the permissions necessary for SRE to troubleshoot AWS resources. Upon assumption of the `ManagedOpenShift-Support-Role`, SRE uses a AWS Security Token Service (STS) to perform troubleshooting actions in your account. SRE can perform multiple actions which include:
+
+* Viewing CloudTrail logs
+* Shutting down a faulty EC2 Instance
+* Creating EC2 snapshots
+
+For a full list of permissions, see sts_support_permission_policy.json in the link:https://docs.openshift.com/rosa/rosa_architecture/rosa-sts-about-iam-resources.html[About IAM resources for ROSA clusters that use STS] user guide.
```
Comment (bmcelvee) [inline modules/policy-identity-access-management.adoc:61]: Just one note - do we have a corresponding link for OSD?

**Comment by bmcelvee** [comment]
Just one link to update and it LGTM! The OSD assembly update it's in the CP stage now so that conflict should be resolved soon!

File: modules/policy-identity-access-management.adoc (hunk 2 above)
Comment (smunje1) [reply modules/policy-identity-access-management.adoc:61]: I am removing the OSD content. OSD doesn't support STS.

Comment (smunje1) [reply modules/policy-identity-access-management.adoc:61]: Also, please see [slack thread](https://redhat-internal.slack.com/archives/C05Q2MLFB0X/p1694093293402269?thread_ts=1694003063.601879&cid=C05Q2MLFB0X)

Cc @karanthshashank @laujohns89

**Pull request merged**